web-console/future-openshift/resource-creation/resource-creation.md

```diff
@@ -0,0 +1,72 @@
+# Resource Creation
+
+## Overview
+* If a resource has a creation form (and YAML), the form will load on default.
+  * After the resource is created, users can edit the resource in YAML only.
+  * In the future, inline edit will be investigated as a way to allow users to edit fields in a form-like manner.
+* Samples available in YAML editor will show YAML code to be copied and pasted as well as a "Use this sample" button that will overwrite the YAML with the sample.
+* Samples available in form will show a "Use this sample" button and will fill in form fields.
+* If users switch from form view to YAML, they will not be able to switch back.
+  * In future, users may be allowed to switch back and forth using a toggle button.
+    * Changes made in YAML will be reflected when switching to form view, and vice versa.
+    * An inline notification should appear on the form view notifying users that any additional fields they add in YAML will be saved.
+
+
+## Pages
```

Comment:
Can we move this section below the designs? Mostly because I can see it quickly becoming inaccurate post 4.2 so seems like the last priority.